libcanard/canard_dsdl.h

```diff
@@ -41,6 +41,16 @@ extern "C" {
 typedef float  CanardDSDLFloat32;
 typedef double CanardDSDLFloat64;
 
+/// Copy the specified number of bits from the source buffer into the destination buffer in accordance with the
```

Comment:
it's not clear what length_bit is the length of; the src array or the dst array? What happens if src is null? dst? What happens if there is a buffer overrun (I'm assuming undefined)?

This also uses size_t which is intended for byte-oriented lengths and isn't appropriate for bit-oriented offsets. This is more of a systemic comment though and doesn't require an action for this PR.

Also,